### PR TITLE
`getPageLength` is missing in `Pagination` interface

### PR DESCRIPTION
**Pull request by ghost**
Can be cherry picked for 5.4, for 5.3 it is not needed, because this component does not exist already

### CODE REVIEW

**Comment by ghost** [comment]
Is used in example code in documentary
https://test54.ilias.de/goto_test54_stys_21_ViewControlPaginationPagination_default_delos.html

File: src/UI/Component/ViewControl/Pagination.php [hunk 1]
```diff
@@ -121,4 +121,9 @@ public function getDropdownLabel(): string;
 	 * will be translated, a custom label will not.
 	 */
 	public function getDefaultDropdownLabel(): string;
+
+	/**
+	 * Calculate the total number of pages.
```
Comment (nhaagen) [inline src/UI/Component/ViewControl/Pagination.php]: +1 for moving this into the interface;
however, the doc was wrong already and should be updated to something like "Get the current number of entries on this page"

**Comment by matthiaskunkel** [comment]
Jour Fixe, 07 OCT 2019 : We highly appreciate this suggestion and accept the PR for trunk.

**Comment by klees** [comment]
Thx @sragfw!

**Pull request merged**